scripts/test/script.py

```diff
@@ -1,13 +1,13 @@
 import subprocess
 from pytest import raises
-
+from six import ensure_str
```

Comment:
could you revise the title of this commit message? "last mile" is not specific enough to denote the change made in this commit.

Reply:
:+1: what is your suggestion?